### ISSUE TITLE
Add mouse button swap for left-handed mode

### ISSUE THREAD

**Issue by missyjcat** [KanoComputing/kano-settings#531]
My kid is left handed so using Kano OS with the current default is limiting for him. Suggest adding one more setting to swap the left/right mouse buttons. I see it's also been asked here in the forum: https://community.kano.me/t/left-handed-mouse-button-swap/410

**Comment by radujipa** [comment]
Hi @missyjcat and thank you for the feedback. Are you using an external mouse by any chance and not a Kano keyboard? 

I've created a feature request internally about this and will work with our Product Owner to prioritise this for a future release.

**Comment by missyjcat** [comment]
Yes, I had an old Raspberry Pi 3 lying around and flashed the Kano OS on it. It's a great OS for kids! But we're using old hardware so just a standard mouse.

I can help if I have the time too. Are you all taking community contributions?

**Comment by radujipa** [comment]
From a `Terminal` could you try using:

Swap left-right mouse buttons:
```
xmodmap -e "pointer = 3 2 1"
```

Restore mouse buttons to default:
```
xmodmap -e "pointer = default"
```

Having had a quick look at this, I think it's going to be a bit more involved to support this system-wide given that we're using a few different technologies. The above configures the button mappings under X11 server, but we'd have to look further into how to integrate that for Qt and Love2D applications (e.g. Dashboard, Story Mode, etc). We are accepting contributions, feel free to look into it.
